src/index.js

```diff
@@ -61,6 +61,7 @@ export default class OasisApp {
     switch (this.versionResponse.major) {
       case 0:
       case 1:
+      case 2:
```

Comment:
is this correct? that version 2 uses the same serialize path implementation?

Reply:
wow github breaks that link by removing the `#`

`https://github.com/Zondax/ledger-oasis/compare/v1.8.2...v2.3.1#diff-1fe0c5c30620ad391d8a90a7580d5995a738548393e38f7329c90429691a0ef6`

reposting in code block

Reply:
oh the 5-component path uses unhardened derivations for the last one?

Reply:
anyway, verified that the new app is compatible. thanks for the reference